### DISCUSSION TITLE
商科领域的ChatPaper

### DISCUSSION THREAD

**Discussion by sangyx** [kaixindelele/ChatPaper#280] [Show and tell]
感谢作者的分享与贡献，我基于这个项目做了一个商科的ChatPaper: <https://github.com/sangyx/deep-business>

这个项目在ChatPaper的基础上做了如下改进：

1. 针对商科主要发表在期刊的特点，对UTD 24的顶刊定制了PDF解析模板，能够解析到更干净的章节内容结构
2. 对论文的introduction及conclusion部分使用预置模板进行总结
3. 允许用户对论文特定章节进行提问

欢迎大家试用，Star!

**Comment by ATang0729** [comment]
好人一生平安
